.github/workflows/osv-scanner.yml

```diff
@@ -26,8 +26,6 @@ jobs:
       scan-args: |-
         --skip-git
         --recursive
-        --config
```

Comment:
Thanks, I missed this in my PR. can you also add `--no-call-analysis=go` along with the TODO comment in line 39?